tests/testthat/test-href.R

```diff
@@ -1,5 +1,8 @@
 context("href")
 
+# This test file fails if run in isolation unless this context is set
+scoped_package_context("test", local_packages = character())
```

Comment:
Can you please add this to the individual tests? It might be better to add a new test "uses local links when remote package equals current" or similar?